pkg/components/storage.go

```diff
@@ -145,9 +144,5 @@ func startCSIPlugin(ctx context.Context, cfg *config.Config, kubeconfigPath stri
 		klog.Warningf("Failed to apply sccs %v: %v", scc, err)
 		return err
 	}
-	if err := assets.ApplyVolumeSnapshotClass(ctx, kubeconfigPath, vcs, nil, nil); err != nil {
```

Comment:
So, now whole `volumeSnapshotClassApplier` and related are unused?

Why do we delete `assets/components/lvms/topolvm_default-volumeclass.yaml`?
Does removing provisioning of thin volume from infra steps cause a failure of MicroShift startup because it cannot apply this file?

Reply:
> So, now whole volumeSnapshotClassApplier and related are unused?

Correct. I opted to leave them for the moment as we may have a need for them later.

> Why do we delete assets/components/lvms/topolvm_default-volumeclass.yaml?
This is a small fix to the early snapshot implementation. Thinpools aren't created automatically, and MicroShift does not check for them, so deploying a volumesnapshotclass object is superfluous and could create some confusion for users.  If a user deploys microshift with snapshotting enabled, they will have handled the thinpool creation, lvmd.yaml configuration, as well as creating a volumesnapclass for the device class.

Reply:
Can that be put into its own commit with a message explaining all of that?